handbook/direction/code-graph/code-insights/index.md

```diff
@@ -1,44 +1,218 @@
+
+There are no other products that can efficiently and effectively provide the deep historical and realtime analysis that Code Insights can provide, largely because any product that wanted to compete with Sourcegraph Code Insights would first have to catch up to the product moat of Sourcegraph Search (which powers Code Insights), and then also build out a Code Insights-like product.
+
+There may be one-off instances of companies that have glued together in-house solutions to answer some of the same problems, but we have already validated (thus far) every one of those companies would prefer to use Code Insights.
+
+While Code Insights is in Beta, you can find a detailed breakdown of "similar" products in the original [Product Document for Code Insights](https://docs.google.com/document/d/1d34gCpt_rUOMAun8phcjNsFofGaaA_N_8znmgaugdKw/edit#bookmark=id.pobwla63lsa7). At some moment pre-GA launch, the product and marketing team will create more standard competitive resources based on the GA product features and merge them into this page.
+
+## Detailed Strategy and Plans
+
+### Strategy for FY22H2:
+
+The next six months are focused on building a product that can quickly prove out its product-market fit via successful customer sales.
+
+### Goals
+
+#### FY22Q3
+
+- OKR 1: We have qualitative/anecdotal evidence that at least 3 customers have a manager/director/VP who is using code insights
+- OKR 2: 50% of customers on 3.31+ make 10 code insights per customer, and have at least 5 weekly unique viewers of insights ([Looker dashboard](https://sourcegraph.looker.com/dashboards-next/209?Latest+Version=))
+
+### Themes (and What's next and why)
+
+#### Powerful but Simple
+
+Our biggest pre-beta learning was that a nice GUI and quick flow to creating a user's first Code Insight dramatically improves initial adoption. As we build a product that needs to be complex and configurable enough to serve the varied needs of our customers and fit perfectly to their codebase, we want to maintain the ease of setting up and navigating Code Insights.
+
+We will:
+
+- Continue abstracting complexity (example: views vs tables) unless necessary to expose it
+- Build new setup features to make it easier to track a variety of other use cases, like with regex capture groups
+- Explore what "quick set up" insights might be possible the first moment a customer uses insights
+
+#### "Tracking" Use Case Needs
+
+A major use case for Code Insights is tracking migrations and similar efforts. There are a number of core features we've learned are valuable to users that we have yet to build.
+
+We will:
+
+- Enable things like custom x-axis time ranges on Insights
+- Allow users to edit insights after creation
+- Explore additional chart visualizations that are helpful
+- Allow engineering leaders to export insights data to their preferred dashboards tools
```

Comment:
This was only listed as a "stretch project" in the Q3 one-pager, while now it uses the strong wording "we will". Is this because this is for FY22H2 and this being a non-stretch goal for FQ4?

Reply:
That is correct!